README.md

```diff
@@ -1,17 +1,31 @@
-# Terraform Ecommerce Microservices on GKE
+# terraform-ecommerce-microservices-on-gke
```

Comment:
The formal name (used by the [solution guide](https://cloud.google.com/architecture/application-development/ecommerce-microservices) and [JSS Console page](https://console.cloud.google.com/products/solutions/catalog)) of this JSS is:
```
Ecommerce web app deployed on Kubernetes
```
Let me know if you'd prefer to use that title here.

Reply:
I personally would prefer title--  What was reasoning behind the repo slug instead?

Reply:
To clarify, there are 3 options:
1.  `terraform-ecommerce-microservices-on-gke` - the name of the repo
2. `Terraform Ecommerce Microservices on GKE` - what the title used to be
3. `Ecommerce web app deployed on Kubernetes` - the formal and final name of the JSS

I want to avoid using **2** because it can cause confusion (not just amongst users, but also amongst Googlers working on JSS 8.1 and 8.2). See [Googler-internal Chat thread here](https://chat.google.com/room/AAAAxW7bZ00/4j1MryC0MG0) and [this Google-internal Doc comment](https://screenshot.googleplex.com/AYYTpoGHMydXhh4.png).

You mentioned:
> I personally would prefer title ...


I assume you're referring to either **2** _or_ **3**, so I will go with **3**. :)